crates/rattler_conda_types/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! `rattler-conda-types` contains data models for types commonly found within the Conda ecosystem.
 //! The library itself doesnt provide any functionality besides parsing the data types.
 
+#[deny(missing_docs)]
```

Comment:
This now only applies to this file. If you want to do this for the whole library use:

```suggestion
#![deny(missing_docs)]
```

Reply:
argh, shoot, I was happy that it looked already pretty decent. but actually more work to do that deserves a seperate PR!